src/overlays/actors/ovl_En_Fd/z_en_fd.c

```diff
@@ -39,11 +40,17 @@ ActorProfile En_Fd_Profile = {
     /**/ EnFd_Draw,
 };
 
+#if OOT_VERSION < NTSC_1_1
+#define SPHERE_DAMAGE 0x00
+#else
+#define SPHERE_DAMAGE 0x04
+#endif
```

Comment:
maybe  `FLARE_DANCER_BODY_DMG` to be a little more specific. shortening to dmg is preference dont really care if its long